### PR TITLE
workflow: Use actions to publish github release with built binaries.

### PR DESCRIPTION
**Pull request by hjpotter92**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,39 +1,104 @@
...
   build:
-    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        # NOTE: Disabling windows and arm64 build environments while
+        # ffmpeg installation issues get resolved.
+        os:
+          - ubuntu-latest
+          - macos-latest
```
Comment (victorges) [inline .github/workflows/build.yml:14]: Maybe better to specify the versions?

**Review by victorges — APPROVED** [review]
LGTM

File: .github/workflows/build.yml (hunk 1 above)
Comment (hjpotter92) [reply .github/workflows/build.yml:14]: yes, i should! thanks

**Comment by hjpotter92** [comment]
> LGTM

any idea about these errors in arm64/windows builds? 
```
 undefined: ErrTranscoderRes
```

appears on local machine too with `GOOS=darwin make`:

```
../../../../../go/pkg/mod/github.com/livepeer/lpms@v0.0.0-20220111151401-0eb91f2facdb/ffmpeg/videoprofile.go:132:16: undefined: ErrTranscoderRes
```

**Pull request merged**